f/flatbuffers/flatbuffers_ubi8.5.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : flatbuffers
+# Version       : v1.12.0
+# Source repo   : https://github.com/google/flatbuffers.git
+# Tested on     : UBI 8.5
+# Language      : C++
+# Travis-Check  : true
+# Script License: Apache License, Version 2 or later
+# Maintainer    : BulkPackageSearch Automation {maintainer}
```

Comment:
Please put you name and email address in maintainer

Reply:
Fixed